docs/getting-started.md

```diff
@@ -0,0 +1,186 @@
+# Getting Started
+
+These instructions assume you use MacOS, and that you are on the internal Broad
+network or the VPN. If the VPN is not installed, follow the instructions
+[at this link](https://broad.io/vpn).
+
+> During this process, you will need your GitHub username, password, and
+personal access token for multiple steps, so make sure to have those handy. If
+you don't have those yet, see the section below, otherwise you can skip to
+[Request Required Access](#2-request-required-access).
+
+## 1. Create a GitHub account
+
+GitHub is where the Broad stores our code and projects. Sign up to this service
+with your **personal** email: https://github.com/join
```

Comment:
I never noticed this before. You can use your personal email, or you could use your broad email. If you use your broad account, then your work GitHub account will be separate from your personal one, which may be what a user wants to do. The important thing is that the GitHub account is linked to your broad account using https://github.broadinstitute.org